docker-compose-core.yml

```diff
@@ -0,0 +1,57 @@
+version: "3.7"
+
+services:
+  harmoni_core:
+    container_name: harmoni_core
+    build:
+      context: .
+      dockerfile: dockerfiles/harmoni/$ROS_DISTRO/core/dockerfile
+      network: host
+    image: harmoniteam/harmoni:$ROS_DISTRO-core
+    init: true
+    environment:
+      DISPLAY: $DISPLAY
+      QT_GRAPHICSSYSTEM: native
+      ROS_DISTRO: $ROS_DISTRO
+      ROS_MASTER_URI: http://172.18.3.4:11311
+      IS_DOCKER_ENV: "true"
+      ROS_HOSTNAME: harmoni_core
+      CATKIN_WS: harmoni_catkin_ws
+    privileged: true
+    networks:
+      ros_net:
+        ipv4_address: 172.18.3.4
+    hostname: harmoni_core
+    ports:
+      - "11312:11312"
+      - "33691:33691"
+      - "8081:8081"
+    devices:
+      - /dev/dri:/dev/dri
+      - /dev/snd:/dev/snd
+      - /dev/video0:/dev/video0
+    volumes:
+      # - harmoni_catkin_ws:/root/harmoni_catkin_ws
+      - ../HARMONI/:/root/local_mount/HARMONI/
+      # Configuration
+      - ~/.aws:/root/.aws/
+      # Other
+      - /tmp/.X11-unix:/tmp/.X11-unix
+      - /etc/timezone:/etc/timezone:ro
+      - /etc/localtime:/etc/localtime:ro
+    working_dir: /root/harmoni_catkin_ws/src/HARMONI
+    command:
+      tail  -f /dev/null
+      #  bash -c "terminator -ue \"echo 'Entering harmoni_core Container... \\n start with roscore and rlharmoniservices' && bash\""
+
+networks:
+  ros_net:
+    driver: bridge
```

Comment:
FYI, with this driver, the host won't be able to communicate with container apps except through the ports noted above ^